<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebCam Drawing Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView_camera"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_buttons">
      <item>
       <widget class="QPushButton" name="btnDetectColor">
        <property name="text">
         <string>Detectează Culoare</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDrawing">
        <property name="text">
         <string>Mod Desenare</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectDrawingColor">
        <property name="text">
         <string>Selectează Culoare Desen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearCanvas">
        <property name="text">
         <string>Șterge Desen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_sliders">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Dimensiune Pensulă:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderBrushSize">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
